doc/reference/instance_options.md

```diff
@@ -67,7 +67,6 @@ Key                                             | Type      | Default
 `cloud-init.network-config`                     | string    | `DHCP on eth0`    | no            | if supported by image     | Network configuration for `cloud-init` (content is used as seed value)
 `cloud-init.user-data`                          | string    | `#cloud-config`   | no            | if supported by image     | User data for `cloud-init` (content is used as seed value)
 `cloud-init.vendor-data`                        | string    | `#cloud-config`   | no            | if supported by image     | Vendor data for `cloud-init` (content is used as seed value)
-`user.meta-data`                                | string    | -                 | no            | if supported by image     | Legacy meta-data for `cloud-init` (content is appended to seed value)
```

Comment:
@ru-fu out of interest, do you know why there isn't a `cloud-init.` equivalent for this one?

Reply:
As far as I understood, it's because there's no reason for using it - the `user.meta-data` was originally added to set the instance name, but that isn't necessary anymore (and also doesn't work).